constants.gradle

```diff
@@ -16,10 +16,10 @@ project.ext {
     releaseVersion = '2.16.1'
     releaseVersionCode = 2016001
     minSdkVersion = 16
-    appTargetSdkVersion = 29
+    appTargetSdkVersion = 31
```

Comment:
This is currently set to 29 on purpose because at least one of the demo apps uses `requestLegacyExternalStorage` as an easy way to access side-loaded media files for testing purposes. So I think we need to update the demo apps to not need that before we can increase this.